### PR TITLE
Corrections from previous releases and blacklisted key bundles

### PR DESCRIPTION
**Pull request by saulmestanza**

### CODE REVIEW

File: iOS-Email-Client/AsyncTasks/SendMailAsyncTask.swift [hunk 1]
```diff
@@ -212,14 +212,26 @@ class SendMailAsyncTask {
...
                 self.setEmailAsFailed()
                 DispatchQueue.main.async {
                     completion(responseData)
                 }
                 return
             }
-            for keys in keysArray {
+            let keyBundles = keysArray["keyBundles"] as! [[String:Any]]
+            let blackListedDevices = keysArray["blacklistedKnownDevices"] as! [[String:Any]]
+            let store: CriptextSessionStore = CriptextSessionStore()
```
Comment (danieltigse) [inline iOS-Email-Client/AsyncTasks/SendMailAsyncTask.swift]: Don't use CriptextSessionStore. Use `DBAxolotl`

File: iOS-Email-Client/Controllers/EmailDetailViewController.swift [hunk 2]
```diff
@@ -201,10 +201,8 @@ extension EmailDetailViewController: UITableViewDelegate, UITableViewDataSource{
         cell.delegate = self
         target = cell.moreOptionsContainerView
         if(isGroupable()){
-            DispatchQueue.main.async {
```
Comment (danieltigse) [inline iOS-Email-Client/Controllers/EmailDetailViewController.swift]: So Mayer's bug is fixed by removing the `DispatchQueue`. What was the original reason of this code?

File: iOS-Email-Client/Controllers/InboxViewController.swift [hunk 3]
```diff
@@ -805,12 +807,14 @@ extension InboxViewController: UITableViewDataSource{
         let thread = mailboxData.threads[indexPath.row]
         
         cell.setFields(thread: thread, label: mailboxData.selectedLabel, myEmail: "\(myAccount.username)\(Constants.domain)")
-        
         if mailboxData.isCustomEditing {
-            if(cell.isSelected){
+            if(selectedCells[indexPath.row] ?? false){
                 cell.setAsSelected()
+                cell.setSelected(true, animated: false)
+                tableView.selectRow(at: indexPath, animated: false, scrollPosition: UITableViewScrollPosition.none)
```
Comment (danieltigse) [inline iOS-Email-Client/Controllers/InboxViewController.swift]: `selectRow` and `setSelected` aren't doing the same?

File: iOS-Email-Client/AsyncTasks/SendMailAsyncTask.swift (hunk 1 above)
Comment (saulmestanza) [reply iOS-Email-Client/AsyncTasks/SendMailAsyncTask.swift]: The class CriptextSessionStore has a method deleteSession which uses DBAxolotl deleteSessionRecord

**Pull request merged**